modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -98,18 +98,20 @@ export default class NestedTabsHtml extends Base {
 		this.setSettings( originalToggleMethods );
 	}
 
-	deactivateActiveTab( tabIndex ) {
+	deactivateActiveTab( newTabIndex ) {
 		const settings = this.getSettings(),
 			activeClass = settings.classes.active,
-			activeTitleFilter = !! tabIndex ? this.getTabTitleFilterSelector( tabIndex ) : settings.ariaAttributes.activeTitleSelector,
```

Comment:
The 'tabIndex' parameter was never used in our code inside the 'deactivateActiveTab()' method.